### ISSUE TITLE
“Heartbeat” failure screen has functionality issues

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#4832]
**Mozilla VPN affected versions:**

* 2.11.0 (2.202210270555);

**Platform tested:**

* Windows;

**Prerequisites:**

* Have the Mozilla VPN app installed and be logged.
* Trigger from the Inspector the heartbeat failure screen;

**Steps to reproduce:**

1. Observe screen behavior;
1. Click the “Try again” button;

**Expected results:**

* The error screen is not dismissed;
* After step 2 the user is redirected to the activation screen;

**Actual results:**

* The screen is dismissed after few seconds;
* After step 2 nothing happens.
!heartb.gif|width=1632,height=948!

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-3178)
┆Reporter: Valentina Virlics
┆Link To Issue: https://mozilla-hub.atlassian.net/browse/VPN-3178

**Comment by data-sync-user** [comment]
➤ Andrea Marchesini commented:

Valentina Virlics Are you able to reproduce it consistently?  What I see in the log is that the captive-portal check timeouts. All the heartbeat tasks are simply waiting to complete the operation.

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Andrea Marchesini On Friday, we reproduce it continuously.

Verified this using the last 2.11.0 (2.202211041406) build from release branch on Windows 10, and it is no longer reproducing.

While clicking the “Try again“ button the user is redirected to activation screen, and not sooner.

Regarding the format of this page, should we keep this issue open, or log another?

!image-20221107-094350.png|width=409,height=741!

We expect to have the button and the link, at the bottom of the screen.

**Comment by data-sync-user** [comment]
➤ Andrea Marchesini commented:

For the front-end bugs, file a separate issue. Thanks!
